robocorp-code/src/robocorp_code/playwright/__playwright__main.py

```diff
@@ -115,18 +125,10 @@ def in_thread():
             # may also be related to VSCode + Windows 11 + Windows Defender + python
             _stdin_write(process, b"\n")
 
-            launched_count = 0
-            for line in iter(process.stdout.readline, ""):
-                contents = line.decode("utf-8", "replace")
-                if not contents:
-                    break
-                full_output.append(contents)
-                if "<launched>" in contents:
-                    launched_count += 1
-                    if launched_count == 2:
-                        # It launches the browser and the 2nd window.
-                        launched_event.set()
+            # launch event timer -> it will release the lock on the launch event
+            launched_event_timer.start()
```

Comment:
I have a question here: didn't `<launched>` appear in the output for you (which is why it needed the change)?